nemo/collections/asr/models/wav2vec/modules/norm.py

```diff
@@ -0,0 +1,32 @@
+from torch import nn
+import torch.nn.functional as F
+
+
+class Fp32GroupNorm(nn.GroupNorm):
```

Comment:
Why do we need these two modules? Can we call the original modules with such params directly?